addons/web_tip/static/src/js/tip.js

```diff
@@ -250,7 +242,13 @@
                         def.resolve();
                     }
                 });
-
+                self.$element.on('mousedown', function($ev) {
+                    if (tip.mode == 'kanban')
+                    {
+                        self.end_tip(tip);
+                        def.resolve();
+                    }
+                });
```

Comment:
@julienlegros binding on .oe_kanban_record not working sometime with mouseup, looks like jquery bug, end selector change for now.